code/modules/hydroponics/hydroponics.dm

```diff
@@ -1110,6 +1115,7 @@
 		name = initial(name)
 		desc = initial(desc)
 		TRAY_NAME_UPDATE
+		particles = NULL
```

Comment:
```suggestion
		particles = null
```

it would be so swag if this compiled

Reply:
This is why you never code in other languages kids